tests/ui/derives/derive-PartialEq-on-non-copy-type.rs

```diff
@@ -0,0 +1,18 @@
+//@ check-pass
```

Comment:
It would be better to add this case to the existing `tests/ui/deriving/deriving-all-codegen.rs` file.

Reply:
Moved, thank you for pointing it out.